### ISSUE TITLE
Issue Tracker

### ISSUE THREAD

**Issue by lmtoan** [amitabhnag/FakeReviewGenerator#2]
Updated May 17, 2018

- [x] 1) Re-organize the main repository and delete the `word-rnn-tensorflow` folder. **Assignee: Amitabh**
- [x] 2) Add in the `demo.py` script to generate review at every epoch. **Assignee: Amitabh**
- [x] 3) Add in the `demo.py` script to smooth "fake" review with Google Translate at every epoch.  **Assignee: Gautam**
- [x] 4) Add in the `demo.py` script to evaluate the "score" of the "fake" review at every epoch.  **Assignee: Toan**
- [x] 5) Fix the default parameters to the most optimal values for the Pitchfork music dataset. **Assignee: Gautam**
- [x] 6) Write `setup.py` script. **Assignee: All**
- [x] 7) Write `requirements.txt` file. **Assignee: All**
- [x] 8) Continuous Integration and generate `requirements.txt` file. **Assignee: All**
- [x] 9) Run `demo.py` on Amazon reviews dataset. **Assignee: All**
- [x] 10) Create a `.ipynb` Notebook to summarize findings and Tensorboard statistics. **Assignee: All**
- [x] 11) Write `README.md` to include all instructions to setup virtual environment and Tensorboard. **Assignee: All**

Updated May 13, 2018

- [x] 1) Test Case for class. Check if the output null or it conatins valid characters. **Assignee: Toan**
- [x] 2) Prepare Amazon food review data. **Assignee: Amitabh**
- [x] 3) Try grammatical check with Amazon food review data.  **Assignee: Gautam**
- [x] 4) Check the translate APIs. Incorporate Google translate APIs as a post processing step. **Assignee: Gautam**
- [x] 5) Add a diagnostics feature with GUI of the loss function. **Assignee: Toan**
- [x] 7) Parameter tuning of the model to get better output. **Assignee: All**

**Comment by lmtoan** [comment]
Backlog:
- [ ] 1) Installation script
- [ ] 2) Project summary on Jupyter Notebook
- [ ] 3) Follow Google documentation guidelines
